core/src/main/java/org/mule/lifecycle/RegistryLifecycleManager.java

```diff
@@ -220,64 +208,4 @@ public void applyCompletedPhases(Object object) throws LifecycleException
         }
     }
 
-    class RegistryLifecycleCallback implements LifecycleCallback<Object>
-    {
-        /**
-         * logger used by this class
-         */
-        protected transient final Log logger = LogFactory.getLog(RegistryLifecycleCallback.class);
-
-        public void onTransition(String phaseName, Object object) throws MuleException
-        {
-            LifecyclePhase phase = phases.get(phaseName);
-
-            if (logger.isDebugEnabled())
```

Comment:
Did not noted that previous comments were related to code you moved. Feel free to disregard them